javascript/ql/test/library-tests/Arrays/arrays.js

```diff
@@ -112,19 +112,19 @@
 
   {  // Test for findLast function
     const list = ["source"];
-    const element = list.findLast((item) => sink(item)); // NOT OK -- Not caught, currently missing dataflow tracking.
+    const element = list.findLast((item) => sink(item)); // NOT OK
     sink(element); // NOT OK
   }
 
   {  // Test for find function
     const list = ["source"];
-    const element = list.find((item) => sink(item)); // NOT OK -- Not caught, currently missing dataflow tracking.
+    const element = list.find((item) => sink(item)); // NOT OK
     sink(element); // NOT OK
   }
 
   {  // Test for findLastIndex function
     const list = ["source"];
-    const element = list.findLastIndex((item) => sink(item)); // NOT OK -- Not caught, currently missing dataflow tracking.
+    const element = list.findLastIndex((item) => sink(item)); // NOT OK
     sink(element); // OK
   }
```

Comment:
Could you also add a test like: 

```
    const element = source.findLastIndex((item) => sink(item)); // NOT OK - only found with taint-tracking. 
    sink(element); // OK
```

Reply:
Adds test case: f1e95a8a1dee4d68038779011bf3414564080f4f